src/model.rs

```diff
@@ -48,6 +48,8 @@ pub struct Collections {
     pub trip_properties: CollectionWithId<TripProperty>,
     pub geometries: CollectionWithId<Geometry>,
     pub admin_stations: Collection<AdminStation>,
+    #[serde(skip)]
+    pub stop_time_headsigns: HashMap<(Idx<VehicleJourney>, u32), String>,
```

Comment:
beacause 
```
error[E0277]: the trait bound `collection::Idx<objects::VehicleJourney>: serde::Serialize` is not satisfied
  --> src/model.rs:51:5
   |
51 |     pub stop_time_headsigns: HashMap<(Idx<VehicleJourney>, u32), String>,
   |     ^^^ the trait `serde::Serialize` is not implemented for `collection::Idx<objects::VehicleJourney>`
   |
   = note: required because of the requirements on the impl of `serde::Serialize` for `(collection::Idx<objects::VehicleJourney>, u32)`
   = note: required because of the requirements on the impl of `serde::Serialize` for `std::collections::HashMap<(collection::Idx<objects::VehicleJourney>, u32), std::string::String>`
   = note: required by `serde::ser::SerializeStruct::serialize_field`
```

and why do I need to serialize it?

Reply:
OK.